Khan/perseus
# Create an demo story show how to integrate the on-screen keypad with ItemRenderer

## Summary:
This story allows us to test the integration of components from math-input with perseus.  The plan is to rebase #340 on top of this PR to demonstrate that using DOM events to communicate between the keypad and input field will also work within perseus as well.

Issue: None

TODO:
- [x] fix styling by getting rid of the padding around document.body

## Test plan:
- yarn start-storybook
- enable mobile emulation in your browser dev tools
- http://localhost:56205/iframe.html?id=perseus-demos-on-screen-keypad--custom-keypad&args=&viewMode=story
- click in the input field, see the keypad animate upwards
- type on the keypad, see that the input field updates accordingly
- tap outside of the keypad and input field to dismiss the keypad

<img width="1072" alt="Screen Shot 2023-01-31 at 11 01 41 AM" src="https://user-images.githubusercontent.com/1044413/215814774-546a7359-e794-4730-9612-9a5811b9ecf1.png">
